teamcity: failed tests on master: testrace/TestGossipAlertsTable, test/TestGossipAlertsTable

The following tests appear to have failed:

[#912639](https://teamcity.cockroachdb.com/viewLog.html?buildId=912639):


```
--- FAIL: test/TestGossipAlertsTable (0.310s)
crdb_internal_test.go:54: SELECT * FROM crdb_internal.gossip_alerts: unexpected multiple results
------- Stdout: -------
W180918 21:02:07.509045 51068 server/status/runtime.go:310  [n?] Could not parse build timestamp: parsing time "" as "2006/01/02 15:04:05": cannot parse "" as "2006"
I180918 21:02:07.517219 51068 server/server.go:848  [n?] monitoring forward clock jumps based on server.clock.forward_jump_check_enabled
I180918 21:02:07.517386 51068 base/addr_validation.go:279  [n?] server certificate addresses: IP=127.0.0.1,::1; DNS=localhost,*.local; CN=node
I180918 21:02:07.517415 51068 base/addr_validation.go:319  [n?] web UI certificate addresses: IP=127.0.0.1,::1; DNS=localhost,*.local; CN=node
I180918 21:02:07.520076 51068 server/config.go:493  [n?] 1 storage engine initialized
I180918 21:02:07.520108 51068 server/config.go:496  [n?] RocksDB cache size: 128 MiB
I180918 21:02:07.520116 51068 server/config.go:496  [n?] store 0: in-memory, size 0 B
I180918 21:02:07.523644 51068 util/stop/stopper.go:537  [n?] quiescing; tasks left:
1      [async] closedts-rangefeed-subscriber
I180918 21:02:07.523871 51068 server/node.go:374  [n?] **** cluster 2822da28-6782-4118-a3b3-32a3c649ac18 has been created
I180918 21:02:07.523894 51068 server/server.go:1412  [n?] **** add additional nodes by specifying --join=127.0.0.1:46077
I180918 21:02:07.524023 51068 gossip/gossip.go:408  [n1] NodeDescriptor set to node_id:1 address:<network_field:"tcp" address_field:"127.0.0.1:46077" > attrs:<> locality:<> ServerVersion:<major_val:2 minor_val:0 patch:0 unstable:13 > build_tag:"v2.2.0-alpha.00000000-913-ga72045b" started_at:1537304527523964389 
I180918 21:02:07.524277 51205 gossip/gossip.go:940  [n1] gossip connectivity
I180918 21:02:07.524909 51068 server/node.go:477  [n1] initialized store [n1,s1]: disk (capacity=512 MiB, available=512 MiB, used=0 B, logicalBytes=7.0 KiB), ranges=1, leases=0, queries=0.00, writes=0.00, bytesPerReplica={p10=7183.00 p25=7183.00 p50=7183.00 p75=7183.00 p90=7183.00 pMax=7183.00}, writesPerReplica={p10=0.00 p25=0.00 p50=0.00 p75=0.00 p90=0.00 pMax=0.00}
I180918 21:02:07.524960 51068 storage/stores.go:242  [n1] read 0 node addresses from persistent storage
I180918 21:02:07.525023 51068 server/node.go:700  [n1] connecting to gossip network to verify cluster ID...
I180918 21:02:07.525046 51068 server/node.go:725  [n1] node connected via gossip and verified as part of cluster "2822da28-6782-4118-a3b3-32a3c649ac18"
I180918 21:02:07.525075 51068 server/node.go:549  [n1] node=1: started with [<no-attributes>=<in-mem>] engine(s) and attributes []
I180918 21:02:07.525296 51068 server/status/recorder.go:611  [n1] available memory from cgroups (8.0 EiB) exceeds system memory 16 GiB, using system memory
I180918 21:02:07.525322 51068 server/server.go:1803  [n1] Could not start heap profiler worker due to: directory to store profiles could not be determined
I180918 21:02:07.525401 51068 server/server.go:1549  [n1] starting https server at 127.0.0.1:45721 (use: 127.0.0.1:45721)
I180918 21:02:07.525422 51068 server/server.go:1551  [n1] starting grpc/postgres server at 127.0.0.1:46077
I180918 21:02:07.525438 51068 server/server.go:1552  [n1] advertising CockroachDB node at 127.0.0.1:46077
I180918 21:02:07.530327 51117 storage/replica_command.go:298  [n1,split,s1,r1/1:/M{in-ax}] initiating a split of this range at key /System/"" [r2]
I180918 21:02:07.536610 51149 storage/replica_command.go:298  [n1,split,s1,r2/1:/{System/-Max}] initiating a split of this range at key /System/NodeLiveness [r3]
I180918 21:02:07.550739 51411 sql/event_log.go:126  [n1,intExec=optInToDiagnosticsStatReporting] Event: "set_cluster_setting", target: 0, info: {SettingName:diagnostics.reporting.enabled Value:true User:root}
I180918 21:02:07.579409 51220 storage/replica_command.go:298  [n1,split,s1,r3/1:/{System/NodeL…-Max}] initiating a split of this range at key /System/NodeLivenessMax [r4]
I180918 21:02:07.590955 51226 storage/replica_command.go:298  [n1,split,s1,r4/1:/{System/NodeL…-Max}] initiating a split of this range at key /System/tsd [r5]
I180918 21:02:07.596747 51406 sql/event_log.go:126  [n1,intExec=set-setting] Event: "set_cluster_setting", target: 0, info: {SettingName:version Value:2.0-13 User:root}
I180918 21:02:07.603325 51232 storage/replica_command.go:298  [n1,split,s1,r5/1:/{System/tsd-Max}] initiating a split of this range at key /System/"tse" [r6]
I180918 21:02:07.610982 51417 sql/event_log.go:126  [n1,intExec=disableNetTrace] Event: "set_cluster_setting", target: 0, info: {SettingName:trace.debug.enable Value:false User:root}
I180918 21:02:07.625063 51437 storage/replica_command.go:298  [n1,split,s1,r6/1:/{System/tse-Max}] initiating a split of this range at key /Table/SystemConfigSpan/Start [r7]
I180918 21:02:07.629396 51470 sql/event_log.go:126  [n1,intExec=initializeClusterSecret] Event: "set_cluster_setting", target: 0, info: {SettingName:cluster.secret Value:6a2f97b4-468d-485a-9af9-b3b394ee11f0 User:root}
I180918 21:02:07.634563 51424 storage/replica_command.go:298  [n1,split,s1,r7/1:/{Table/System…-Max}] initiating a split of this range at key /Table/11 [r8]
W180918 21:02:07.634917 51422 storage/intent_resolver.go:668  [n1,s1] failed to push during intent resolution: failed to push "sql txn" id=f3777235 key=/Table/SystemConfigSpan/Start rw=true pri=0.00640480 iso=SERIALIZABLE stat=PENDING epo=0 ts=1537304527.632399773,0 orig=1537304527.632399773,0 max=1537304527.632399773,0 wto=false rop=false seq=2
I180918 21:02:07.640613 51454 storage/replica_command.go:298  [n1,split,s1,r8/1:/{Table/11-Max}] initiating a split of this range at key /Table/12 [r9]
I180918 21:02:07.644765 51450 sql/event_log.go:126  [n1,intExec=create-default-db] Event: "create_database", target: 50, info: {DatabaseName:defaultdb Statement:CREATE DATABASE IF NOT EXISTS defaultdb User:root}
I180918 21:02:07.650732 51478 sql/event_log.go:126  [n1,intExec=create-default-db] Event: "create_database", target: 51, info: {DatabaseName:postgres Statement:CREATE DATABASE IF NOT EXISTS postgres User:root}
I180918 21:02:07.652237 51068 server/server.go:1605  [n1] done ensuring all necessary migrations have run
I180918 21:02:07.652266 51068 server/server.go:1608  [n1] serving sql connections
I180918 21:02:07.655112 51532 sql/event_log.go:126  [n1] Event: "node_join", target: 1, info: {Descriptor:{NodeID:1 Address:{NetworkField:tcp AddressField:127.0.0.1:46077} Attrs: Locality: ServerVersion:2.0-13 BuildTag:v2.2.0-alpha.00000000-913-ga72045b StartedAt:1537304527523964389 LocalityAddress:[]} ClusterID:2822da28-6782-4118-a3b3-32a3c649ac18 StartedAt:1537304527523964389 LastUp:1537304527523964389}
I180918 21:02:07.655931 51530 server/server_update.go:67  [n1] no need to upgrade, cluster already at the newest version
I180918 21:02:07.656119 51496 storage/replica_command.go:298  [n1,split,s1,r9/1:/{Table/12-Max}] initiating a split of this range at key /Table/13 [r10]
I180918 21:02:07.658328 51068 server/testserver.go:446  had 9 ranges at startup, expected 22
I180918 21:02:07.658740 51068 server/testserver.go:446  had 9 ranges at startup, expected 22
I180918 21:02:07.661297 51547 storage/replica_command.go:298  [n1,split,s1,r10/1:/{Table/13-Max}] initiating a split of this range at key /Table/14 [r11]
I180918 21:02:07.662577 51068 server/testserver.go:446  had 10 ranges at startup, expected 22
I180918 21:02:07.662878 51068 server/testserver.go:446  had 10 ranges at startup, expected 22
I180918 21:02:07.672196 51551 storage/replica_command.go:298  [n1,split,s1,r11/1:/{Table/14-Max}] initiating a split of this range at key /Table/15 [r12]
I180918 21:02:07.674895 51068 server/testserver.go:446  had 11 ranges at startup, expected 22
I180918 21:02:07.675186 51068 server/testserver.go:446  had 11 ranges at startup, expected 22
I180918 21:02:07.675406 51068 server/testserver.go:446  had 11 ranges at startup, expected 22
I180918 21:02:07.675621 51068 server/testserver.go:446  had 11 ranges at startup, expected 22
I180918 21:02:07.675834 51068 server/testserver.go:446  had 11 ranges at startup, expected 22
I180918 21:02:07.676032 51068 server/testserver.go:446  had 11 ranges at startup, expected 22
I180918 21:02:07.676256 51068 server/testserver.go:446  had 11 ranges at startup, expected 22
I180918 21:02:07.676465 51068 server/testserver.go:446  had 11 ranges at startup, expected 22
I180918 21:02:07.676708 51068 server/testserver.go:446  had 11 ranges at startup, expected 22
I180918 21:02:07.676938 51068 server/testserver.go:446  had 11 ranges at startup, expected 22
I180918 21:02:07.677117 51068 server/testserver.go:446  had 11 ranges at startup, expected 22
I180918 21:02:07.677280 51068 server/testserver.go:446  had 11 ranges at startup, expected 22
I180918 21:02:07.677558 51068 server/testserver.go:446  had 11 ranges at startup, expected 22
I180918 21:02:07.681016 51489 storage/replica_command.go:298  [n1,split,s1,r12/1:/{Table/15-Max}] initiating a split of this range at key /Table/16 [r13]
I180918 21:02:07.696296 51068 server/testserver.go:446  had 13 ranges at startup, expected 22
I180918 21:02:07.697065 51068 server/testserver.go:446  had 13 ranges at startup, expected 22
I180918 21:02:07.697195 51515 storage/replica_command.go:298  [n1,split,s1,r13/1:/{Table/16-Max}] initiating a split of this range at key /Table/17 [r14]
I180918 21:02:07.701035 51605 storage/replica_command.go:298  [n1,split,s1,r14/1:/{Table/17-Max}] initiating a split of this range at key /Table/18 [r15]
I180918 21:02:07.702843 51068 server/testserver.go:446  had 14 ranges at startup, expected 22
I180918 21:02:07.703755 51068 server/testserver.go:446  had 14 ranges at startup, expected 22
I180918 21:02:07.705011 51068 server/testserver.go:446  had 14 ranges at startup, expected 22
I180918 21:02:07.706722 51620 storage/replica_command.go:298  [n1,split,s1,r15/1:/{Table/18-Max}] initiating a split of this range at key /Table/19 [r16]
I180918 21:02:07.711482 51068 server/testserver.go:446  had 16 ranges at startup, expected 22
I180918 21:02:07.711963 51642 storage/replica_command.go:298  [n1,split,s1,r16/1:/{Table/19-Max}] initiating a split of this range at key /Table/20 [r17]
I180918 21:02:07.717462 51068 server/testserver.go:446  had 17 ranges at startup, expected 22
I180918 21:02:07.718029 51627 storage/replica_command.go:298  [n1,split,s1,r17/1:/{Table/20-Max}] initiating a split of this range at key /Table/21 [r18]
I180918 21:02:07.723881 51651 storage/replica_command.go:298  [n1,split,s1,r18/1:/{Table/21-Max}] initiating a split of this range at key /Table/22 [r19]
I180918 21:02:07.726076 51068 server/testserver.go:446  had 18 ranges at startup, expected 22
I180918 21:02:07.728360 51656 storage/replica_command.go:298  [n1,split,s1,r19/1:/{Table/22-Max}] initiating a split of this range at key /Table/23 [r20]
I180918 21:02:07.735848 51658 storage/replica_command.go:298  [n1,split,s1,r20/1:/{Table/23-Max}] initiating a split of this range at key /Table/50 [r21]
I180918 21:02:07.745423 51699 storage/replica_command.go:298  [n1,split,s1,r21/1:/{Table/50-Max}] initiating a split of this range at key /Table/51 [r22]
I180918 21:02:07.746763 51068 server/testserver.go:446  had 21 ranges at startup, expected 22
I180918 21:02:07.782314 51068 util/stop/stopper.go:537  quiescing; tasks left:
1      [async] closedts-rangefeed-subscriber
--- FAIL: testrace/TestGossipAlertsTable (4.960s)

------- Stdout: -------
W180918 21:12:15.481235 43944 server/status/runtime.go:310  [n?] Could not parse build timestamp: parsing time "" as "2006/01/02 15:04:05": cannot parse "" as "2006"
I180918 21:12:15.579657 43944 server/server.go:848  [n?] monitoring forward clock jumps based on server.clock.forward_jump_check_enabled
I180918 21:12:15.580360 43944 base/addr_validation.go:279  [n?] server certificate addresses: IP=127.0.0.1,::1; DNS=localhost,*.local; CN=node
I180918 21:12:15.580461 43944 base/addr_validation.go:319  [n?] web UI certificate addresses: IP=127.0.0.1,::1; DNS=localhost,*.local; CN=node
I180918 21:12:15.586175 43944 server/config.go:493  [n?] 1 storage engine initialized
I180918 21:12:15.586252 43944 server/config.go:496  [n?] RocksDB cache size: 128 MiB
I180918 21:12:15.586316 43944 server/config.go:496  [n?] store 0: in-memory, size 0 B
I180918 21:12:15.773043 43944 server/node.go:374  [n?] **** cluster 61f554d5-6b45-4e5a-95c1-eb43c194f058 has been created
I180918 21:12:15.773280 43944 server/server.go:1412  [n?] **** add additional nodes by specifying --join=127.0.0.1:38115
I180918 21:12:15.774760 43944 gossip/gossip.go:408  [n1] NodeDescriptor set to node_id:1 address:<network_field:"tcp" address_field:"127.0.0.1:38115" > attrs:<> locality:<> ServerVersion:<major_val:2 minor_val:0 patch:0 unstable:13 > build_tag:"v2.2.0-alpha.00000000-913-ga72045b" started_at:1537305135774234993 
I180918 21:12:15.777290 44004 gossip/gossip.go:940  [n1] gossip connectivity
I180918 21:12:15.833242 43944 server/node.go:477  [n1] initialized store [n1,s1]: disk (capacity=512 MiB, available=512 MiB, used=0 B, logicalBytes=7.0 KiB), ranges=1, leases=1, queries=0.00, writes=0.00, bytesPerReplica={p10=7183.00 p25=7183.00 p50=7183.00 p75=7183.00 p90=7183.00 pMax=7183.00}, writesPerReplica={p10=0.00 p25=0.00 p50=0.00 p75=0.00 p90=0.00 pMax=0.00}
I180918 21:12:15.833962 43944 storage/stores.go:242  [n1] read 0 node addresses from persistent storage
I180918 21:12:15.840330 43944 server/node.go:700  [n1] connecting to gossip network to verify cluster ID...
I180918 21:12:15.840676 43944 server/node.go:725  [n1] node connected via gossip and verified as part of cluster "61f554d5-6b45-4e5a-95c1-eb43c194f058"
I180918 21:12:15.841120 43944 server/node.go:549  [n1] node=1: started with [<no-attributes>=<in-mem>] engine(s) and attributes []
I180918 21:12:15.858147 43944 server/status/recorder.go:611  [n1] available memory from cgroups (8.0 EiB) exceeds system memory 16 GiB, using system memory
I180918 21:12:15.858353 43944 server/server.go:1803  [n1] Could not start heap profiler worker due to: directory to store profiles could not be determined
I180918 21:12:15.867636 43944 server/server.go:1549  [n1] starting https server at 127.0.0.1:44233 (use: 127.0.0.1:44233)
I180918 21:12:15.867873 43944 server/server.go:1551  [n1] starting grpc/postgres server at 127.0.0.1:38115
I180918 21:12:15.867938 43944 server/server.go:1552  [n1] advertising CockroachDB node at 127.0.0.1:38115
W180918 21:12:15.868316 43944 jobs/registry.go:315  [n1] unable to get node liveness: node not in the liveness table
I180918 21:12:15.877122 43791 storage/replica_command.go:298  [n1,split,s1,r1/1:/M{in-ax}] initiating a split of this range at key /System/"" [r2]
I180918 21:12:16.124264 44008 storage/replica_command.go:298  [n1,split,s1,r2/1:/{System/-Max}] initiating a split of this range at key /System/NodeLiveness [r3]
I180918 21:12:16.439731 44006 sql/event_log.go:126  [n1,intExec=optInToDiagnosticsStatReporting] Event: "set_cluster_setting", target: 0, info: {SettingName:diagnostics.reporting.enabled Value:true User:root}
I180918 21:12:16.452704 44200 storage/replica_command.go:298  [n1,split,s1,r3/1:/{System/NodeL…-Max}] initiating a split of this range at key /System/NodeLivenessMax [r4]
I180918 21:12:16.744243 44171 storage/replica_command.go:298  [n1,split,s1,r4/1:/{System/NodeL…-Max}] initiating a split of this range at key /System/tsd [r5]
I180918 21:12:16.999810 44244 storage/replica_command.go:298  [n1,split,s1,r5/1:/{System/tsd-Max}] initiating a split of this range at key /System/"tse" [r6]
I180918 21:12:17.095260 44097 rpc/nodedialer/nodedialer.go:92  [n1,consistencyChecker,s1,r1/1:/{Min-System/}] connection to n1 established
I180918 21:12:17.219168 44088 sql/event_log.go:126  [n1,intExec=set-setting] Event: "set_cluster_setting", target: 0, info: {SettingName:version Value:2.0-13 User:root}
I180918 21:12:17.317587 44295 storage/replica_command.go:298  [n1,split,s1,r6/1:/{System/tse-Max}] initiating a split of this range at key /Table/SystemConfigSpan/Start [r7]
I180918 21:12:17.565682 44313 storage/replica_command.go:298  [n1,split,s1,r7/1:/{Table/System…-Max}] initiating a split of this range at key /Table/11 [r8]
W180918 21:12:17.581998 44311 storage/intent_resolver.go:668  [n1,s1] failed to push during intent resolution: failed to push "unnamed" id=05ea3e66 key=/Table/SystemConfigSpan/Start rw=true pri=0.00101854 iso=SERIALIZABLE stat=PENDING epo=0 ts=1537305137.238812520,0 orig=1537305137.238812520,0 max=1537305137.238812520,0 wto=false rop=false seq=6
I180918 21:12:17.594994 44004 gossip/gossip.go:940  [n1] gossip connectivity
  n1 [sentinel];
I180918 21:12:17.743048 44291 sql/event_log.go:126  [n1,intExec=disableNetTrace] Event: "set_cluster_setting", target: 0, info: {SettingName:trace.debug.enable Value:false User:root}
I180918 21:12:17.801407 44223 storage/replica_command.go:298  [n1,split,s1,r8/1:/{Table/11-Max}] initiating a split of this range at key /Table/12 [r9]
I180918 21:12:17.880250 44237 storage/replica_command.go:298  [n1,split,s1,r9/1:/{Table/12-Max}] initiating a split of this range at key /Table/13 [r10]
I180918 21:12:18.044979 44372 storage/replica_command.go:298  [n1,split,s1,r10/1:/{Table/13-Max}] initiating a split of this range at key /Table/14 [r11]
I180918 21:12:18.180356 44327 storage/replica_command.go:298  [n1,split,s1,r11/1:/{Table/14-Max}] initiating a split of this range at key /Table/15 [r12]
I180918 21:12:18.214391 44388 sql/event_log.go:126  [n1,intExec=initializeClusterSecret] Event: "set_cluster_setting", target: 0, info: {SettingName:cluster.secret Value:c95d4028-508f-492f-97d2-b71d893a4946 User:root}
I180918 21:12:18.332777 44394 storage/replica_command.go:298  [n1,split,s1,r12/1:/{Table/15-Max}] initiating a split of this range at key /Table/16 [r13]
I180918 21:12:18.483709 44319 sql/event_log.go:126  [n1,intExec=create-default-db] Event: "create_database", target: 50, info: {DatabaseName:defaultdb Statement:CREATE DATABASE IF NOT EXISTS defaultdb User:root}
I180918 21:12:18.530965 44331 storage/replica_command.go:298  [n1,split,s1,r13/1:/{Table/16-Max}] initiating a split of this range at key /Table/17 [r14]
I180918 21:12:18.617444 44345 sql/event_log.go:126  [n1,intExec=create-default-db] Event: "create_database", target: 51, info: {DatabaseName:postgres Statement:CREATE DATABASE IF NOT EXISTS postgres User:root}
I180918 21:12:18.677373 44399 storage/replica_command.go:298  [n1,split,s1,r14/1:/{Table/17-Max}] initiating a split of this range at key /Table/18 [r15]
I180918 21:12:18.734720 43944 server/server.go:1605  [n1] done ensuring all necessary migrations have run
I180918 21:12:18.737096 43944 server/server.go:1608  [n1] serving sql connections
I180918 21:12:18.763465 43944 server/testserver.go:446  had 14 ranges at startup, expected 22
I180918 21:12:18.773161 43944 server/testserver.go:446  had 14 ranges at startup, expected 22
I180918 21:12:18.787147 43944 server/testserver.go:446  had 14 ranges at startup, expected 22
I180918 21:12:18.791649 43944 server/testserver.go:446  had 14 ranges at startup, expected 22
I180918 21:12:18.793189 43944 server/testserver.go:446  had 14 ranges at startup, expected 22
I180918 21:12:18.794303 43944 server/testserver.go:446  had 14 ranges at startup, expected 22
I180918 21:12:18.795327 43944 server/testserver.go:446  had 14 ranges at startup, expected 22
I180918 21:12:18.796300 43944 server/testserver.go:446  had 14 ranges at startup, expected 22
I180918 21:12:18.802814 44271 sql/event_log.go:126  [n1] Event: "node_join", target: 1, info: {Descriptor:{NodeID:1 Address:{NetworkField:tcp AddressField:127.0.0.1:38115} Attrs: Locality: ServerVersion:2.0-13 BuildTag:v2.2.0-alpha.00000000-913-ga72045b StartedAt:1537305135774234993 LocalityAddress:[]} ClusterID:61f554d5-6b45-4e5a-95c1-eb43c194f058 StartedAt:1537305135774234993 LastUp:1537305135774234993}
I180918 21:12:18.811526 44269 server/server_update.go:67  [n1] no need to upgrade, cluster already at the newest version
I180918 21:12:18.877662 44272 storage/replica_command.go:298  [n1,split,s1,r15/1:/{Table/18-Max}] initiating a split of this range at key /Table/19 [r16]
I180918 21:12:18.963121 43944 server/testserver.go:446  had 15 ranges at startup, expected 22
I180918 21:12:19.009954 44404 storage/replica_command.go:298  [n1,split,s1,r16/1:/{Table/19-Max}] initiating a split of this range at key /Table/20 [r17]
I180918 21:12:19.034887 43944 server/testserver.go:446  had 16 ranges at startup, expected 22
I180918 21:12:19.036318 43944 server/testserver.go:446  had 16 ranges at startup, expected 22
I180918 21:12:19.037555 43944 server/testserver.go:446  had 16 ranges at startup, expected 22
I180918 21:12:19.038691 43944 server/testserver.go:446  had 16 ranges at startup, expected 22
I180918 21:12:19.039604 43944 server/testserver.go:446  had 16 ranges at startup, expected 22
I180918 21:12:19.040599 43944 server/testserver.go:446  had 16 ranges at startup, expected 22
I180918 21:12:19.041418 43944 server/testserver.go:446  had 16 ranges at startup, expected 22
I180918 21:12:19.042071 43944 server/testserver.go:446  had 16 ranges at startup, expected 22
I180918 21:12:19.042731 43944 server/testserver.go:446  had 16 ranges at startup, expected 22
I180918 21:12:19.043777 43944 server/testserver.go:446  had 16 ranges at startup, expected 22
I180918 21:12:19.045177 43944 server/testserver.go:446  had 16 ranges at startup, expected 22
I180918 21:12:19.065292 43944 server/testserver.go:446  had 16 ranges at startup, expected 22
I180918 21:12:19.067735 43944 server/testserver.go:446  had 16 ranges at startup, expected 22
I180918 21:12:19.071233 43944 server/testserver.go:446  had 16 ranges at startup, expected 22
I180918 21:12:19.077045 43944 server/testserver.go:446  had 16 ranges at startup, expected 22
I180918 21:12:19.115695 44407 storage/replica_command.go:298  [n1,split,s1,r17/1:/{Table/20-Max}] initiating a split of this range at key /Table/21 [r18]
I180918 21:12:19.166698 43944 server/testserver.go:446  had 17 ranges at startup, expected 22
I180918 21:12:19.185156 43944 server/testserver.go:446  had 17 ranges at startup, expected 22
I180918 21:12:19.265969 44401 storage/replica_command.go:298  [n1,split,s1,r18/1:/{Table/21-Max}] initiating a split of this range at key /Table/22 [r19]
I180918 21:12:19.275076 43944 server/testserver.go:446  had 18 ranges at startup, expected 22
I180918 21:12:19.354490 44443 storage/replica_command.go:298  [n1,split,s1,r19/1:/{Table/22-Max}] initiating a split of this range at key /Table/23 [r20]
I180918 21:12:19.386255 43944 server/testserver.go:446  had 19 ranges at startup, expected 22
I180918 21:12:19.488218 44491 storage/replica_command.go:298  [n1,split,s1,r20/1:/{Table/23-Max}] initiating a split of this range at key /Table/50 [r21]
I180918 21:12:19.562614 43944 server/testserver.go:446  had 20 ranges at startup, expected 22
I180918 21:12:19.631235 44475 storage/replica_command.go:298  [n1,split,s1,r21/1:/{Table/50-Max}] initiating a split of this range at key /Table/51 [r22]
I180918 21:12:19.852994 43944 util/stop/stopper.go:537  quiescing; tasks left:
1      [async] transport racer
1      [async] closedts-rangefeed-subscriber
I180918 21:12:19.867042 43944 util/stop/stopper.go:537  quiescing; tasks left:
1      [async] transport racer
I180918 21:12:19.853410 44196 kv/transport_race.go:91  transport race promotion: ran 76 iterations on up to 799 requestscrdb_internal_test.go:54: SELECT * FROM crdb_internal.gossip_alerts: unexpected multiple results
------- Stdout: -------
W180918 21:02:07.509045 51068 server/status/runtime.go:310  [n?] Could not parse build timestamp: parsing time "" as "2006/01/02 15:04:05": cannot parse "" as "2006"
I180918 21:02:07.517219 51068 server/server.go:848  [n?] monitoring forward clock jumps based on server.clock.forward_jump_check_enabled
I180918 21:02:07.517386 51068 base/addr_validation.go:279  [n?] server certificate addresses: IP=127.0.0.1,::1; DNS=localhost,*.local; CN=node
I180918 21:02:07.517415 51068 base/addr_validation.go:319  [n?] web UI certificate addresses: IP=127.0.0.1,::1; DNS=localhost,*.local; CN=node
I180918 21:02:07.520076 51068 server/config.go:493  [n?] 1 storage engine initialized
I180918 21:02:07.520108 51068 server/config.go:496  [n?] RocksDB cache size: 128 MiB
I180918 21:02:07.520116 51068 server/config.go:496  [n?] store 0: in-memory, size 0 B
I180918 21:02:07.523644 51068 util/stop/stopper.go:537  [n?] quiescing; tasks left:
1      [async] closedts-rangefeed-subscriber
I180918 21:02:07.523871 51068 server/node.go:374  [n?] **** cluster 2822da28-6782-4118-a3b3-32a3c649ac18 has been created
I180918 21:02:07.523894 51068 server/server.go:1412  [n?] **** add additional nodes by specifying --join=127.0.0.1:46077
I180918 21:02:07.524023 51068 gossip/gossip.go:408  [n1] NodeDescriptor set to node_id:1 address:<network_field:"tcp" address_field:"127.0.0.1:46077" > attrs:<> locality:<> ServerVersion:<major_val:2 minor_val:0 patch:0 unstable:13 > build_tag:"v2.2.0-alpha.00000000-913-ga72045b" started_at:1537304527523964389 
I180918 21:02:07.524277 51205 gossip/gossip.go:940  [n1] gossip connectivity
I180918 21:02:07.524909 51068 server/node.go:477  [n1] initialized store [n1,s1]: disk (capacity=512 MiB, available=512 MiB, used=0 B, logicalBytes=7.0 KiB), ranges=1, leases=0, queries=0.00, writes=0.00, bytesPerReplica={p10=7183.00 p25=7183.00 p50=7183.00 p75=7183.00 p90=7183.00 pMax=7183.00}, writesPerReplica={p10=0.00 p25=0.00 p50=0.00 p75=0.00 p90=0.00 pMax=0.00}
I180918 21:02:07.524960 51068 storage/stores.go:242  [n1] read 0 node addresses from persistent storage
I180918 21:02:07.525023 51068 server/node.go:700  [n1] connecting to gossip network to verify cluster ID...
I180918 21:02:07.525046 51068 server/node.go:725  [n1] node connected via gossip and verified as part of cluster "2822da28-6782-4118-a3b3-32a3c649ac18"
I180918 21:02:07.525075 51068 server/node.go:549  [n1] node=1: started with [<no-attributes>=<in-mem>] engine(s) and attributes []
I180918 21:02:07.525296 51068 server/status/recorder.go:611  [n1] available memory from cgroups (8.0 EiB) exceeds system memory 16 GiB, using system memory
I180918 21:02:07.525322 51068 server/server.go:1803  [n1] Could not start heap profiler worker due to: directory to store profiles could not be determined
I180918 21:02:07.525401 51068 server/server.go:1549  [n1] starting https server at 127.0.0.1:45721 (use: 127.0.0.1:45721)
I180918 21:02:07.525422 51068 server/server.go:1551  [n1] starting grpc/postgres server at 127.0.0.1:46077
I180918 21:02:07.525438 51068 server/server.go:1552  [n1] advertising CockroachDB node at 127.0.0.1:46077
I180918 21:02:07.530327 51117 storage/replica_command.go:298  [n1,split,s1,r1/1:/M{in-ax}] initiating a split of this range at key /System/"" [r2]
I180918 21:02:07.536610 51149 storage/replica_command.go:298  [n1,split,s1,r2/1:/{System/-Max}] initiating a split of this range at key /System/NodeLiveness [r3]
I180918 21:02:07.550739 51411 sql/event_log.go:126  [n1,intExec=optInToDiagnosticsStatReporting] Event: "set_cluster_setting", target: 0, info: {SettingName:diagnostics.reporting.enabled Value:true User:root}
I180918 21:02:07.579409 51220 storage/replica_command.go:298  [n1,split,s1,r3/1:/{System/NodeL…-Max}] initiating a split of this range at key /System/NodeLivenessMax [r4]
I180918 21:02:07.590955 51226 storage/replica_command.go:298  [n1,split,s1,r4/1:/{System/NodeL…-Max}] initiating a split of this range at key /System/tsd [r5]
I180918 21:02:07.596747 51406 sql/event_log.go:126  [n1,intExec=set-setting] Event: "set_cluster_setting", target: 0, info: {SettingName:version Value:2.0-13 User:root}
I180918 21:02:07.603325 51232 storage/replica_command.go:298  [n1,split,s1,r5/1:/{System/tsd-Max}] initiating a split of this range at key /System/"tse" [r6]
I180918 21:02:07.610982 51417 sql/event_log.go:126  [n1,intExec=disableNetTrace] Event: "set_cluster_setting", target: 0, info: {SettingName:trace.debug.enable Value:false User:root}
I180918 21:02:07.625063 51437 storage/replica_command.go:298  [n1,split,s1,r6/1:/{System/tse-Max}] initiating a split of this range at key /Table/SystemConfigSpan/Start [r7]
I180918 21:02:07.629396 51470 sql/event_log.go:126  [n1,intExec=initializeClusterSecret] Event: "set_cluster_setting", target: 0, info: {SettingName:cluster.secret Value:6a2f97b4-468d-485a-9af9-b3b394ee11f0 User:root}
I180918 21:02:07.634563 51424 storage/replica_command.go:298  [n1,split,s1,r7/1:/{Table/System…-Max}] initiating a split of this range at key /Table/11 [r8]
W180918 21:02:07.634917 51422 storage/intent_resolver.go:668  [n1,s1] failed to push during intent resolution: failed to push "sql txn" id=f3777235 key=/Table/SystemConfigSpan/Start rw=true pri=0.00640480 iso=SERIALIZABLE stat=PENDING epo=0 ts=1537304527.632399773,0 orig=1537304527.632399773,0 max=1537304527.632399773,0 wto=false rop=false seq=2
I180918 21:02:07.640613 51454 storage/replica_command.go:298  [n1,split,s1,r8/1:/{Table/11-Max}] initiating a split of this range at key /Table/12 [r9]
I180918 21:02:07.644765 51450 sql/event_log.go:126  [n1,intExec=create-default-db] Event: "create_database", target: 50, info: {DatabaseName:defaultdb Statement:CREATE DATABASE IF NOT EXISTS defaultdb User:root}
I180918 21:02:07.650732 51478 sql/event_log.go:126  [n1,intExec=create-default-db] Event: "create_database", target: 51, info: {DatabaseName:postgres Statement:CREATE DATABASE IF NOT EXISTS postgres User:root}
I180918 21:02:07.652237 51068 server/server.go:1605  [n1] done ensuring all necessary migrations have run
I180918 21:02:07.652266 51068 server/server.go:1608  [n1] serving sql connections
I180918 21:02:07.655112 51532 sql/event_log.go:126  [n1] Event: "node_join", target: 1, info: {Descriptor:{NodeID:1 Address:{NetworkField:tcp AddressField:127.0.0.1:46077} Attrs: Locality: ServerVersion:2.0-13 BuildTag:v2.2.0-alpha.00000000-913-ga72045b StartedAt:1537304527523964389 LocalityAddress:[]} ClusterID:2822da28-6782-4118-a3b3-32a3c649ac18 StartedAt:1537304527523964389 LastUp:1537304527523964389}
I180918 21:02:07.655931 51530 server/server_update.go:67  [n1] no need to upgrade, cluster already at the newest version
I180918 21:02:07.656119 51496 storage/replica_command.go:298  [n1,split,s1,r9/1:/{Table/12-Max}] initiating a split of this range at key /Table/13 [r10]
I180918 21:02:07.658328 51068 server/testserver.go:446  had 9 ranges at startup, expected 22
I180918 21:02:07.658740 51068 server/testserver.go:446  had 9 ranges at startup, expected 22
I180918 21:02:07.661297 51547 storage/replica_command.go:298  [n1,split,s1,r10/1:/{Table/13-Max}] initiating a split of this range at key /Table/14 [r11]
I180918 21:02:07.662577 51068 server/testserver.go:446  had 10 ranges at startup, expected 22
I180918 21:02:07.662878 51068 server/testserver.go:446  had 10 ranges at startup, expected 22
I180918 21:02:07.672196 51551 storage/replica_command.go:298  [n1,split,s1,r11/1:/{Table/14-Max}] initiating a split of this range at key /Table/15 [r12]
I180918 21:02:07.674895 51068 server/testserver.go:446  had 11 ranges at startup, expected 22
I180918 21:02:07.675186 51068 server/testserver.go:446  had 11 ranges at startup, expected 22
I180918 21:02:07.675406 51068 server/testserver.go:446  had 11 ranges at startup, expected 22
I180918 21:02:07.675621 51068 server/testserver.go:446  had 11 ranges at startup, expected 22
I180918 21:02:07.675834 51068 server/testserver.go:446  had 11 ranges at startup, expected 22
I180918 21:02:07.676032 51068 server/testserver.go:446  had 11 ranges at startup, expected 22
I180918 21:02:07.676256 51068 server/testserver.go:446  had 11 ranges at startup, expected 22
I180918 21:02:07.676465 51068 server/testserver.go:446  had 11 ranges at startup, expected 22
I180918 21:02:07.676708 51068 server/testserver.go:446  had 11 ranges at startup, expected 22
I180918 21:02:07.676938 51068 server/testserver.go:446  had 11 ranges at startup, expected 22
I180918 21:02:07.677117 51068 server/testserver.go:446  had 11 ranges at startup, expected 22
I180918 21:02:07.677280 51068 server/testserver.go:446  had 11 ranges at startup, expected 22
I180918 21:02:07.677558 51068 server/testserver.go:446  had 11 ranges at startup, expected 22
I180918 21:02:07.681016 51489 storage/replica_command.go:298  [n1,split,s1,r12/1:/{Table/15-Max}] initiating a split of this range at key /Table/16 [r13]
I180918 21:02:07.696296 51068 server/testserver.go:446  had 13 ranges at startup, expected 22
I180918 21:02:07.697065 51068 server/testserver.go:446  had 13 ranges at startup, expected 22
I180918 21:02:07.697195 51515 storage/replica_command.go:298  [n1,split,s1,r13/1:/{Table/16-Max}] initiating a split of this range at key /Table/17 [r14]
I180918 21:02:07.701035 51605 storage/replica_command.go:298  [n1,split,s1,r14/1:/{Table/17-Max}] initiating a split of this range at key /Table/18 [r15]
I180918 21:02:07.702843 51068 server/testserver.go:446  had 14 ranges at startup, expected 22
I180918 21:02:07.703755 51068 server/testserver.go:446  had 14 ranges at startup, expected 22
I180918 21:02:07.705011 51068 server/testserver.go:446  had 14 ranges at startup, expected 22
I180918 21:02:07.706722 51620 storage/replica_command.go:298  [n1,split,s1,r15/1:/{Table/18-Max}] initiating a split of this range at key /Table/19 [r16]
I180918 21:02:07.711482 51068 server/testserver.go:446  had 16 ranges at startup, expected 22
I180918 21:02:07.711963 51642 storage/replica_command.go:298  [n1,split,s1,r16/1:/{Table/19-Max}] initiating a split of this range at key /Table/20 [r17]
I180918 21:02:07.717462 51068 server/testserver.go:446  had 17 ranges at startup, expected 22
I180918 21:02:07.718029 51627 storage/replica_command.go:298  [n1,split,s1,r17/1:/{Table/20-Max}] initiating a split of this range at key /Table/21 [r18]
I180918 21:02:07.723881 51651 storage/replica_command.go:298  [n1,split,s1,r18/1:/{Table/21-Max}] initiating a split of this range at key /Table/22 [r19]
I180918 21:02:07.726076 51068 server/testserver.go:446  had 18 ranges at startup, expected 22
I180918 21:02:07.728360 51656 storage/replica_command.go:298  [n1,split,s1,r19/1:/{Table/22-Max}] initiating a split of this range at key /Table/23 [r20]
I180918 21:02:07.735848 51658 storage/replica_command.go:298  [n1,split,s1,r20/1:/{Table/23-Max}] initiating a split of this range at key /Table/50 [r21]
I180918 21:02:07.745423 51699 storage/replica_command.go:298  [n1,split,s1,r21/1:/{Table/50-Max}] initiating a split of this range at key /Table/51 [r22]
I180918 21:02:07.746763 51068 server/testserver.go:446  had 21 ranges at startup, expected 22
I180918 21:02:07.782314 51068 util/stop/stopper.go:537  quiescing; tasks left:
1      [async] closedts-rangefeed-subscriber

```

Please assign, take a look and update the issue accordingly.

--- a/pkg/sql/crdb_internal_test.go
+++ b/pkg/sql/crdb_internal_test.go
@@ -49,7 +49,7 @@ func TestGossipAlertsTable(t *testing.T) {
 	}
 
 	ie := s.InternalExecutor().(*sql.InternalExecutor)
-	row, err := ie.QueryRow(ctx, "test", nil /* txn */, "SELECT * FROM crdb_internal.gossip_alerts")
+	row, err := ie.QueryRow(ctx, "test", nil /* txn */, "SELECT * FROM crdb_internal.gossip_alerts WHERE store_id = 123")
 	if err != nil {
 		t.Fatal(err)
 	}

